projects/scuba-physics/src/lib/performanceTests.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { Time } from './Time';
+import { AlgorithmParams, BuhlmannAlgorithm } from './BuhlmannAlgorithm';
+import { Gas, Gases } from './Gases';
+import { Segment, Segments } from './Segments';
+import { OptionExtensions } from './Options.spec';
+import { Salinity } from './pressure-converter';
+import { Options, SafetyStop } from './Options';
+import { StandardGases } from './StandardGases';
+import { Precision } from './precision';
+import { FeatureFlags } from './featureFlags';
+import { Tank } from './Tanks';
+import { Diver } from './Diver';
+import { DepthConverter } from './depth-converter';
+import { Consumption, ConsumptionOptions } from './consumption';
+
+
+export function calculatePlanFor(gases: Gases, source: Segments, options: Options): Segment[] {
+    const algorithm = new BuhlmannAlgorithm();
+    const parameters = AlgorithmParams.forMultilevelDive(source, gases, options);
+    const decoPlan = algorithm.decompression(parameters);
+    return decoPlan.segments;
+}
+
+describe('Buhlmann Algorithm - Performance', () => {
+    const options = OptionExtensions.createOptions(0.4, 0.85, 1.4, 1.6, Salinity.salt);
+    options.safetyStop = SafetyStop.always;
+
+    beforeEach(() => {
+        options.safetyStop = SafetyStop.always;
+        options.salinity = Salinity.salt;
+        options.roundStopsToMinutes = true;
+        options.decoStopDistance = 3;
+        options.altitude = 0;
+    });
+
+    it('for trimix 75 m deep dive is calculated within 400 ms', () => {
+        FeatureFlags.Instance.collectSaturation = true;
```

Comment:
If setting this to true, it needs to be reset in shared test setup, otherwise all following tests will have this flag.